protocol/examples/messageData/election_setup.json

```diff
@@ -1,19 +1,19 @@
 {
     "object": "election",
     "action": "setup",
-    "id": "XXX",
-    "lao": "XXX",
-    "name": "XXX",
-    "version": "XXX",
-    "created_at": 123,
-    "start_time": 123,
-    "end_time": 123,
+    "id": "JVehtpaFJwPc2dp8VuiskXwxmjTQGT6OiIVbei-EbBI=",
+    "lao": "fzJSZjKf-2cbXH7kds9H8NORuuFIRLkevJlN7qQemjo=",
+    "name": "Election",
+    "version": "1.0.0",
+    "created_at": 1633098941,
```

Comment:
Do we have a convention in the project on when to use "created_at" instead of "creation" (like in lao_create.json) ? Maybe the couple "begin"/"end" is clearer for the naming.